gulpy/bch.py

```diff
@@ -44,9 +44,23 @@ def process(data, sesh, diag):
             log.error("Failed to insert {}".format(o), exc_info=True)
             failures += 1
 
+        #commit current insertions once every batch_size insertions.
```

Comment:
Please follow [PEP 8 code style](https://www.python.org/dev/peps/pep-0008/#id31): Space after `#`; correct capitalization for sentences..